### ISSUE TITLE
Multiple runs per file not read

### ISSUE THREAD

**Issue by thomasarmstrong** [cta-observatory/pyeventio#132]
Just ran into the problem that the new SimTelEventSource in ctapipe, using pyeventio, cannot read multiple runs in one file (e.g. produced with read_hess using --dst-file and --dst-level 0 flags on multiple simtel files). 
If a file contains, for example, runs 1 and 2, it will read the events in run 1 and then finish.

**Comment by dneise** [comment]
Hello Thomas, thanks for reporting! Can you upload a small example file?

**Comment by thomasarmstrong** [comment]
[run1011_merged.dst0.simtel.gz](https://github.com/cta-observatory/pyeventio/files/2791803/run1011_merged.dst0.simtel.gz) (13Mb, 2 runs, ~300 events each)

For example if I were to run: 

```python 
from ctapipe.io import EventSourceFactory
source = EventSourceFactory.produce(input_url='run1011_merged.dst0.simtel.gz')
for event in source:
    print(event.r1.obs_id)
```
I will just receive run 10, if however I try:
```python
from ctapipe.io.hessioeventsource import HESSIOEventSource
source = HESSIOEventSource(input_url='run1011_merged.dst0.simtel.gz')
for event in source:
    print(event.r1.obs_id)
```
I obtain both run 10 and 11

**Comment by dneise** [comment]
Hey Thomas. If you have time, could you test if this version already fulfills your usecase?

https://github.com/cta-observatory/pyeventio/archive/experiment_with_multiple_runs.tar.gz

you should be able to install it with (in your cta conda environment):

    pip install https://github.com/cta-observatory/pyeventio/archive/experiment_with_multiple_runs.tar.gz

You'll get a (red) warning saying:

    ctapipe 0.6.1.post101 has requirement eventio==0.11.0, but you'll have eventio 0.14.1 which is incompatible.

just ignore that for the moment.

You should be able to revert back to the normal version by:

    pip install eventio==0.11.0

**Comment by dneise** [comment]
Cool! So next steps would be some code revision for the PR #133 and once that is merged and v14.1 is published, we would need to get ctapipe to use v14.1.
Honestly I am not sure why ctapipe is currently using v11.0

**Comment by MaxNoe** [comment]
During the sprint we decided to fix the version of pyeventio for ctapipe and there was no reason to update since then.
